src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -55,9 +55,32 @@ <h3 ng-if="$ctrl.isInsideAnotherModal" class="signup-header" translate>Create Yo
       </div>
     </div>
 
-    <loading ng-if="$ctrl.isLoading" class="okta-signup-loading">
-      <div ng-if="$ctrl.currentStep === 5" class="loading-message" translate>OKTA_REGISTERING</div>
-    </loading>
+    <loading ng-if="$ctrl.isLoading && $ctrl.currentStep !== 5" class="okta-signup-loading"></loading>
+
+    <div ng-if="$ctrl.currentStep === 5" class="welcome-message">
+      <loading class="redirect-loading" inline="true" icon-first="true" iconFirst="true">
+        <div class="loading-message" translate>Redirecting in {{$ctrl.redirectToOktaCountdown}}</div>
+      </loading>
+      <h2 translate>Welcome, {{$ctrl.scope.firstName}}</h2>
```

Comment:
I'm not sure that this is going to work for non-English languages. I think we'll need to put `Welcome, {{firstName}}` in a label in `app.config.js` and pass in the variables as described here: https://angular-translate.github.io/docs/#/guide/06_variable-replacement

Reply:
I went ahead and fixed this